bokeh/models/glyphs.py

```diff
@@ -893,6 +926,39 @@ class Text(Glyph):
     The %s values for the text.
     """)
 
+class VBar(Glyph):
+    """ Render vertical bars, given a center coordinate, width and (top, bottom) coordinates. """
+
+    __example__ = "tests/glyphs/VBar.py"
+
+    # a canonical order for positional args that can be used for any
+    # functions derived from this class
+    _args = ('x', 'width', 'top', 'bottom')
+
+    x = NumberSpec(help="""
+    The x-coordinates of the centers of the vertical bars.
+    """)
+
+    width = NumberSpec(help="""
+    The widths of the vertical bars.
+    """)
+
+    bottom = NumberSpec(help="""
+    The y-coordinates of the bottom edges.
+    """, default=0)
```

Comment:
OK, being super picky here, but every other property puts the help last, I think ti would be good to be consistent, so:
```
+    bottom = NumberSpec(default=0, help="""
 +    The y-coordinates of the bottom edges.
 +    """)
 
```